src/handlers.ts

```diff
@@ -30,6 +30,19 @@ export const generateApiKeyHandler = (): express.RequestHandler => {
     return handler;
 };
 
+export const fetchOtcPriceHandler = async (quoter: Quoter, req: express.Request, res: express.Response) => {
+    const takerRequestResponse = parseTakerRequest(req);
+
+    if (!takerRequestResponse.isValid) {
+        return res.status(HttpStatus.BAD_REQUEST).json({ errors: takerRequestResponse.errors });
+    }
+
+    const response = await quoter.fetchIndicativeOtcQuoteAsync(takerRequestResponse.takerRequest);
```

Comment:
do we need a try/catch around this?

Reply:
express will handle this, so should be ok - however, I just found something that I missed